event/listener.php

```diff
@@ -221,4 +227,36 @@ public function show_idea($event)
 			'FORUM_NAME'		=> $this->language->lang('IDEAS'),
 		));
 	}
+
+	/**
+	 * Show users as viewing Ideas on Who Is Online page
+	 *
+	 * @param object $event The event object
+	 * @return null
+	 * @access public
+	 */
+	public function viewonline_ideas($event)
+	{
+		if ($event['on_page'][1] == 'app')
+		{
+			if (strrpos($event['row']['session_page'], 'app.' . $this->php_ext . '/ideas/post') === 0)
+			{
+				$event['location'] = $this->language->lang('POSTING_NEW_IDEA');
+				$event['location_url'] = $this->helper->route('phpbb_ideas_index_controller');
+			}
+			else if (strrpos($event['row']['session_page'], 'app.' . $this->php_ext . '/ideas') === 0)
```

Comment:
By the way, worst case, if this doesn't work on dot-com, then it will probably just display the user is on index page, which is exactly what Titania currently does since it doesn't have this functionality to it ;)